mk/PcapPlusPlus.mk.dpdk

```diff
@@ -1,13 +1,35 @@
 ### DPDK ###
 
+PKGCONF = pkg-config
+
 # includes
+TMPI = $(shell $(PKGCONF) --cflags-only-I libdpdk 2>/dev/null)
+TMPFLAG = $(shell $(PKGCONF) --cflags-only-other libdpdk 2>/dev/null)
+
+ifeq ($(TMPI),)
 PCAPPP_INCLUDES += -I$(RTE_SDK)/build/include
+else
+PCAPPP_INCLUDES += $(TMPI)
+endif
 
 # libs dir
+TMPLD = $(shell $(PKGCONF) --libs-l libdpdk 2>/dev/null)
```

Comment:
I'm not sure the `--libs-l` flag exists, isn't it `--libs-only-l`?

Reply:
Sorry about this, the 2>/dev/null hid the typo :(